ckanext/oidc_pkce/templates/user/login.html

```diff
@@ -4,11 +4,11 @@
 #}
 
 {% block form %}
-    <p><a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-primary"><i class="fa fa-sign-in"></i> Log in via single sign-on</a></p>
+    <p><a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-primary"><i class="fa fa-sign-in"></i> Sign in with your Digital ID</a></p>
     {{ super() }}
 {% endblock %}
 
 {% block help_register_button %}
     {{ super() }}
-    <a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-secondary">Register via single sign-on</a>
+    Learn more about using a <a href="https://www.qld.gov.au/qdifaq">Digital ID</a>
```

Comment:
have an helper in the plugin to control adding the 'register' text as an if condition.


also add a block so we inject our own info text via our theme here.

{% block help_register_info_text %}
{% endblock %}